### PR TITLE
Updated instructions to match GCP interface

### LINKED ISSUE #351: Cloud Backup instructions for Google Drive are out of sync with GCP interface
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/docs/blob/master/CONTRIBUTING.md
- [x] I have searched the existing issues and I am convinced that mine is new.

**Describe the issue**

In following the instructions for setting up Cloud Backup for Google Drive, the instructions do not match the current interface for Google Cloud Platform. Specifically, the steps for creating a service account have changed.

**Suggestions**

After the step to enable the Google Drive API, 

- [x] Replace the "Click credentials button" step with a step to click the "Credentials" menu on the left-side of the screen
- [x] Replace the "Click the service account link" step with a step to click the "Manage Service Accounts" link towards the bottom of the screen
- [x] Update screenshots to reflect changes, including highlighting relevant screen items

### PR DESCRIPTION
**Pull request by adamhogle**
Resolves #351.

### CODE REVIEW

**Comment by fichtner** [comment]
CC @Greelan (can't add you as reviewer without adding you to org, maybe you would like an invite?)

**Comment by Greelan** [comment]
Sure, happy to review 😀

**Comment by fichtner** [comment]
@Greelan much better, thanks :D

File: source/manual/how-tos/cloud_backup.rst [hunk 1]
```diff
@@ -84,11 +84,11 @@ First you need to set up a project in the Google developer console.
 
     .. image:: ./images/google_drive_enable.png
 
--  On the next page that appears, click the **CREATE CREDENTIALS** button:
+-  On the next page that appears, click the **Credentials** item on the left-side menu:
 
     .. image:: ./images/google_create_credentials.png
```
Comment (Greelan) [inline source/manual/how-tos/cloud_backup.rst:90]: I think these two steps can be simplified by just directing the user directly to the service account page. So first delete the Credentials page step (the corresponding image `google_create_credentials.png` can also be deleted).

```suggestion
```

File: source/manual/how-tos/cloud_backup.rst [hunk 2]
```diff
@@ -84,11 +84,11 @@ First you need to set up a project in the Google developer console.
...
 
--  The Credentials Wizard will then appear. Click on the **service account** hyperlink that appears on the page:
+-  On the Credentials page, click the **Manage service accounts** hyperlink that appears next to the Service Account table:
```
Comment (Greelan) [inline source/manual/how-tos/cloud_backup.rst:91]: Then direct the user to search for "service accounts" and go directly to the page. I have attached a replacement screenshot for `google_select_service_account.png`, which will need to be substituted: google_select_service_account).

```suggestion
-  Now create a service account for your project. In the search bar at the top of the page, search for “service accounts" and click on the corresponding search result:

```

File: source/manual/how-tos/cloud_backup.rst [hunk 3]
```diff
@@ -119,7 +119,7 @@ First you need to set up a project in the Google developer console.
 -  You will be prompted to download the key. Do so, and then click **CLOSE** in the popup screen.
 
 
-2. Set up a Google Drive folder
+1. Set up a Google Drive folder
```
Comment (Greelan) [inline source/manual/how-tos/cloud_backup.rst:122]: This change is not correct, as creating the Google Drive folder is the second step. The first step is creating the Google project.

```suggestion
2. Set up a Google Drive folder
```

**Review by Greelan — CHANGES_REQUESTED** [review]
Thanks very much for taking the time to suggest updates. It's unfortunate that Google seems to like playing with its UI a lot!

I've suggested some changes to your PR with the aim of making it simpler.

File: source/manual/how-tos/cloud_backup.rst (hunk 1 above)
Comment (adamhogle) [reply source/manual/how-tos/cloud_backup.rst]: Done.

File: source/manual/how-tos/cloud_backup.rst (hunk 2 above)
Comment (adamhogle) [reply source/manual/how-tos/cloud_backup.rst]: I agree, this is a much easier way to get to this, and it should change less in the future.

File: source/manual/how-tos/cloud_backup.rst (hunk 3 above)
Comment (adamhogle) [reply source/manual/how-tos/cloud_backup.rst]: This was an unintended change that I missed when preparing the PR. I've reverted the code.

**Review by Greelan — APPROVED** [review]
Looks good, thanks!

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**